include/envoy/ssl/connection.h

```diff
@@ -15,33 +15,39 @@ class Connection {
   virtual ~Connection() {}
 
   /**
-   * @return whether the peer certificate is presented.
+   * @return bool whether the peer certificate is presented.
    **/
   virtual bool peerCertificatePresented() const PURE;
 
   /**
-   * @return the URI in the SAN feld of the local certificate. Returns "" if there is no local
-   *         certificate, or no SAN field, or no URI.
+   * @return std::string the URI in the SAN feld of the local certificate. Returns "" if there is no
+   *local certificate, or no SAN field, or no URI.
```

Comment:
super nit: missing space after '*'. Same in several other places below.

Reply:
So I added std::string from the style nit above to these doc strings, and it made the line too long.  So I ran the style fix script, and that is what formatted these comments like this.  I can go manually add the extra space(I agree, it's more aesthetic), but we may want to get it in the fix script, or at least not have the fix script do this when the column width is too large on the previous line.